lib/feature/question/presentation/pages/setting.dart

```diff
@@ -74,26 +70,23 @@ class _SettingPageState extends State<SettingPage> {
                           );
                         }).toList(),
                       )
-
                     ],
                   ),
                   Row(
                     children: [
                       Padding(
                         padding: const EdgeInsets.all(24.0),
                         child: Container(
-                          child:
-                          Text(
-                              "ONLY WEEK QUESTION"
-                          ),
+                          child: Text("ONLY WEEK QUESTION"),
                         ),
                       ),
-                      Switch(value: _values[0],
-                        onChanged: (value){
-                          setState((){
+                      Switch(
+                        value: _values[0],
+                        onChanged: (value) {
+                          setState(() {
```

Comment:
in clean architecture, you don't need to use setState any more, whole this stuff should be moved to bloc. Bloc is responsible of any logic in your page. Page should be dummy and be responsible only on widgets and how to display them

Reply:
Oh,,,, I got it.
I've just understand that I can't escape from Bloc!!!!!

I'll see name-game again and learn bloc.😄